### DISCUSSION TITLE
Lesson 1: What is the difference between unit and integer?

### DISCUSSION THREAD

**Discussion by vicshi06** [smartcontractkit/full-blockchain-solidity-course-py#662] [Q&A] [answered]
Hi,

I am just very curious. What really is the difference between unit and integer in solidity?

**Comment by vicshi06** [comment, marked as answer]
damn it I just realized it is uint not unit.

**Comment by cromewar** [comment]
Well you basically answered your question, uint is the default internet unit for solidity.
